doc_classes/OpenXRFbPassthroughGeometry.xml

```diff
@@ -1,15 +1,21 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="OpenXRFbPassthroughGeometry" inherits="Node3D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/godotengine/godot/master/doc/class.xsd">
 	<brief_description>
+		A mesh that passthrough imagery will be rendered upon (via the [code]XR_FB_passthrough[/code] extension).
 	</brief_description>
 	<description>
+		A mesh that passthrough imagery will be rendered upon (via the [code]XR_FB_passthrough[/code] extension).
+		Even though the node can be placed anywhere in 3D space, everything rendered by Godot is always shown on top of passthrough. If you want to give the impression that objects rendered by Godot can move behind the passthrough, then you must set [member enable_hole_punch] to [code]true[/code].
```

Comment:
Hole punch is enabled by default for this node, since I think it's the more common use case. Personally, this line gives me the impression that it's not. Maybe the wording could be improved here, saying that it's enabled by default.

Reply:
Ah, I forgot that it's enabled by default. I think I'll just remove this bit - there's another description of hole punching on the `enable_hole_punch` property, which is probably sufficient.